test/turnt.toml

```diff
@@ -1,11 +1,16 @@
+[envs.chop_setup]
+binary = true
+command = "python3 ../slow_odgi/chop_by.py < {filename}"
+output.n = "-"
+
 [envs.chop_oracle]
 binary = true
-command = "odgi chop -i {filename} -c 3 -o - | odgi view -g -i - | python3 ../slow_odgi/mygfa.py --nl"
+command = "odgi chop -i {filename} -c $(cat {base}.n) -o - | odgi view -g -i - | python3 ../slow_odgi/mygfa.py --nl"
```

Comment:
It occurs to me that you could avoid the use of the "setup file" here by just invoking the relevant command inline. Something like:

```py
command = "odgi chop -i {filename} -c `python3 ../slow_odgi/chop_by.py < {filename}` -o - | odgi view -g -i - | python3 ../slow_odgi/mygfa.py --nl"
```

That would come at the expense of "re-doing" this calculation every time, but maybe that's not so bad.